src/include/migraphx/matcher.hpp

```diff
@@ -784,7 +784,7 @@ auto skip_broadcasts_converts(Ms... ms)
 }
 
 template <class T>
-inline auto has_value(T x, float tolerance = 1e-6)
+inline auto has_value(T x, float tolerance = 1e-12)
```

Comment:
I dont think we should change the default parameter, only for the specific `add_0` matcher, you can pass a different tolerance `has_value(0.0f, 1e-12)`.

Reply:
Oh didnt realize that. Thanks I can fix this